authority/authority.go

```diff
@@ -111,3 +119,8 @@ func (a *Authority) init() error {
 
 	return nil
 }
+
+// Shutdown safely shuts down any clients, databases, etc. held by the Authority.
+func (a *Authority) Shutdown() error {
+	return a.db.Shutdown()
+}
```

Comment:
We are not calling this method, so we are not shutting down properly the db.

Reply:
Yea, I just added this last night when I was trying to figure out why my DB wasn't syncing offline updates (turned out I was being stupid and using different db file locations). Anyway, I still need to call this from other places.

Reply:
Just add the call [here](https://github.com/smallstep/certificates/blob/master/ca/ca.go#L123)